engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeStdlibTest.scala

```diff
@@ -202,6 +204,17 @@ class RuntimeStdlibTest
     }
     suggestions.isEmpty shouldBe false
 
+    // check that the library is indexed
+    val stdlibSuggestions = responses.collect {
+      case Api.Response(
+            None,
+            Api.SuggestionsDatabaseModuleUpdateNotification(file, _, as, xs)
+          ) if file.getPath.contains("Vector") =>
+        (xs.nonEmpty || as.nonEmpty) shouldBe true
+    }
```

Comment:
I would suggest checking somewhere in the update, if possible, that this is in fact `Standard.Base`.

What if `Vector` at some point gets into the Builtins? Maybe unlikely, but checking for the library name instead would be clearer and more robust.